src/main/java/io/personium/engine/source/FsServiceResourceSourceManager.java

```diff
@@ -212,15 +214,24 @@ private void parseServiceTag() {
             doc = builder.parse(is);
             Element el = doc.getDocumentElement();
             this.serviceSubject = el.getAttribute("subject");
+            IFilenameResolver resolver = new FilenameResolverByRoute();
             NodeList nl = doc.getElementsByTagNameNS("*", "path");
-            for (int i = 0; i < nl.getLength(); i++) {
-                NamedNodeMap nnm = nl.item(i).getAttributes();
-                pathMap.put(nnm.getNamedItem("name").getNodeValue(), nnm.getNamedItem("src").getNodeValue());
+            try {
+                for (int i = 0; i < nl.getLength(); i++) {
+                    NamedNodeMap nnm = nl.item(i).getAttributes();
+                    resolver.registerRoute(nnm.getNamedItem("name").getNodeValue(), nnm.getNamedItem("src").getNodeValue());
+                }
+                this.srcResolver = resolver;
+            } catch (RouteRegistrationException e) {
+                log.error("RouteRegistrationException is thrown", e.getInternalException());
```

Comment:
- `log.error` is too high
- better to log name and src information
- better to give original exception to log it's stacktrace

How about changing to the following?

```java
log.info("Route registration failed. (name=XXX, src=YYY)", e);
```

Reply:
As we discussed,

- include name and src information to RouteRegistrationException and it's message
- no logging in this exception